### PR TITLE
Plat 639 adjust login cookies -- tests fix

### PR DESCRIPTION
**Pull request by nour-borgi**

### CODE REVIEW

File: test/spec/controllers/login.js [hunk 1]
```diff
@@ -85,19 +76,19 @@ describe('Controller: LoginCtrl', function () {
...
-      httpBackend.expectGET(new RegExp('.*/authenticate/incorrect@user.org'))
+      httpBackend.when('POST', new RegExp('.*/authenticate/local')).respond(401)
 
       createController()
 
       scope.loginEmail = 'incorrect@user.org'
       scope.loginPassword = 'incorrect-password'
       scope.validateLogin()
-
+
```
Comment (bradsawadye) [inline test/spec/controllers/login.js:86]:
```suggestion
 
```

File: test/spec/controllers/login.js [hunk 2]
```diff
@@ -85,19 +76,19 @@ describe('Controller: LoginCtrl', function () {
...
       scope.loginPassword = 'incorrect-password'
       scope.validateLogin()
-
+      
       // One error should exist - 'Busy checking login credentials'
       scope.alerts.login.length.should.equal(1)
       scope.alerts.login[0].type.should.equal('warning')
       scope.alerts.login[0].msg.should.equal('Busy checking your credentials...')
-
+
```
Comment (bradsawadye) [inline test/spec/controllers/login.js:91]:
```suggestion

```

File: test/spec/controllers/login.js [hunk 3]
```diff
@@ -137,14 +127,15 @@ describe('Controller: LoginCtrl', function () {
...
   })
 
   describe('*resetRootPassword() tests', function () {
     it('should run the resetRootPassword() function return error for not all fields being supplied', function () {
-      httpBackend.expectGET(new RegExp('.*/authenticate/root@openhim.org'))
-      httpBackend.expectGET(new RegExp('.*/users/root@openhim.org'))
+      httpBackend.when('POST', new RegExp('.*/authenticate/local')).respond(201)
+
+      // httpBackend.expectGET(new RegExp('.*/authenticate/local'))
+      // httpBackend.expectGET(new RegExp('.*/users/root@openhim.org'))
```
Comment (bradsawadye) [inline test/spec/controllers/login.js:138]: Should these not be removed?

File: test/spec/controllers/profile.js [hunk 4]
```diff
@@ -116,8 +122,9 @@ describe('Controller: ProfileCtrl', function () {
   })
 
   it('should test for all validation and return TRUE - hasErrors', function () {
+    httpBackend.expectGET(new RegExp('.*/me'))
     createController()
-
+
```
Comment (bradsawadye) [inline test/spec/controllers/profile.js:127]:
```suggestion

```

File: test/spec/services/login.js [hunk 5]
```diff
@@ -12,31 +12,30 @@ describe('Service: login', function () {
...
-    httpBackend.when('GET', new RegExp('.*/authenticate/.*')).respond({
-      salt: 'test-salt',
-      ts: new Date(new Date().getTime() + 3600000).toISOString() // 1 hour ahead
+    httpBackend.when('GET', new RegExp('.*/me')).respond(404)
+    
+    httpBackend.when('POST', new RegExp('.*/authenticate/local')).respond({
+      body: "User Authenticated successfully"
     })
-
+
```
Comment (bradsawadye) [inline test/spec/services/login.js:26]:
```suggestion

```

File: test/spec/controllers/login.js (hunk 1 above)
Comment (nour-borgi) [reply test/spec/controllers/login.js]: done

File: test/spec/controllers/login.js (hunk 3 above)
Comment (nour-borgi) [reply test/spec/controllers/login.js]: yes cool, done

**Review by bradsawadye — APPROVED** [review]

**Pull request merged**